src/drive/web/modules/drive/Passphrase.jsx

```diff
@@ -13,22 +16,55 @@ class Passphrase extends Component {
   constructor(props) {
     super(props)
     this.state = {
-      passphrase: ''
+      passphrase: '',
+      encryptedVault: null
     }
   }
 
+  async getEncryptedVault() {
+    const { client } = this.props
+    const settings = await client.query(
+      client.find('io.cozy.settings').getById('io.cozy.settings.instance')
+    )
+    let encVault
```

Comment:
Naming: `encVault` is an encrypted key rather than a vault, no ? Imho you look into the a clear vault for the right encrypted key.